hoot-core/src/main/cpp/hoot/core/io/OsmJosmXmlChangesetFileWriter.cpp

```diff
@@ -82,13 +82,13 @@ void OsmJosmXmlChangesetFileWriter::_writeXmlActionAttribute(QXmlStreamWriter& w
   QString action;
   switch (_change.getType())
   {
-  case Change::ChangeType::Create:
   case Change::ChangeType::NoChange:
     //  Create doesn't require an 'action' attribute
     return;
   case Change::ChangeType::Delete:
     action = "delete";
     break;
+  case Change::ChangeType::Create:
```

Comment:
@bmarchant I realized JOSM needs this change in behavior to recognize the new features as needing upload (i.e. enable the upload button when the layer is loaded)